src/traits/networking/libp2p_network.rs

```diff
@@ -572,21 +542,24 @@ impl<
         }
     }
 
-    #[instrument(name = "Libp2pNetwork::message_node", skip_all)]
-    async fn message_node(
-        &self,
-        message: Message<TYPES, LEAF, PROPOSAL>,
-        recipient: TYPES::SignatureKey,
-    ) -> Result<(), NetworkError> {
+    // TODO make non-blocking
+    // TODO make requestid meaningful (e.g. track it)
```

Comment:
I'm curious about why we would want to track `RequestId`. Is it for testing purposes?

Reply:
I was thinking to be able to report the status of the request. E.g. to answer the following sorts of questions: was the request completed? Is the request in the process of being cancelled? Has the request already been cancelled?

Reply:
Yeah makes sense!